packages/app-store/routing-forms/pages/router/[...appPages].tsx

```diff
@@ -14,7 +14,7 @@ export default function Router({ form, message }: inferSSRProps<typeof getServer
       </Head>
       <div className="mx-auto my-0 max-w-3xl md:my-24">
         <div className="w-full max-w-4xl ltr:mr-2 rtl:ml-2">
-          <div className="bg-default -mx-4 rounded-sm border border-neutral-200 p-4 py-6 sm:mx-0 sm:px-8">
+          <div className="text-default bg-default -mx-4 rounded-sm border border-neutral-200 p-4 py-6 sm:mx-0 sm:px-8">
```

Comment:
It fixes the issue of text being black in black theme and not visible.

The issue was only present in Embed because embed doesn't use color-scheme(but non-embed app uses it)